### ISSUE TITLE
Use TYPE (instead of USER_TYPE) as USER property

### ISSUE THREAD

**Issue by oweste** [Titan-Systems/titan#92]
It looks like the new USER property [TYPE = PERSON | SERVICE | LEGACY_SERVICE | NULL](https://docs.snowflake.com/en/sql-reference/sql/create-user#label-user-type-property)
have been [added to titan, but as a non-existing property called USER_TYPE.](https://github.com/Titan-Systems/titan/pull/87/files#diff-f4d727185e5b63c3b96a3aceb655b2862eea4597cf1dcd2a44a4ad5b72aa0c47)

I would expect this YAML to work:
```
users:
  - name: some_user
    owner: USERADMIN
    email: some.user@example.com
    user: PERSON
```

**Comment by teej** [comment]
I try to stay 1:1 to the original SQL when possible. Since "type" has semantic meaning in Python I made a judgement call to use "user_type" in this case. I can make this more clear in the docs.

**Comment by oweste** [comment]
What I noticed, when I used the user_type as a YMAL argument, like this:
```
users:
  - name: some_user
    owner: USERADMIN
    email: some.user@example.com
    user_type: PERSON
```
... was that this was the SQL sent to Snowflake:
```
ALTER USER SOME_USER SET user_type = 'PERSON'
```
So, I guess some last-mile mapping from the in-parameter to the generated SQL-parameter is missing then.

**Comment by teej** [comment]
That's a bug, I'll get it fixed.

**Comment by schlich** [comment]
@teej I ran into this bug as well!

**Comment by teej** [comment]
Fixed in [v0.9](https://github.com/Titan-Systems/titan/releases/tag/v0.9.1)
